src/web/routes/application/confirm.js

```diff
@@ -1,11 +1,30 @@
+const httpStatus = require('http-status-codes')
+const { path, isNil } = require('ramda')
 const { handleRequestForPath } = require('./middleware')
 const { CONFIRM_URL, ELIGIBLE } = require('./common/constants')
+const { wrapError } = require('./common/formatters')
 
-const getConfirmPage = (req, res) => {
+const toPounds = pence => (parseInt(pence, 10) / 100).toFixed(2)
+
+const isNilOrLessThanZero = value => isNil(value) || value <= 0
```

Comment:
Sorry for being picky, but this is `isNilOrLessThanOrEqualToZero`, you forgot the equality.